provisioning_templates/PXELinux/rancheros_pxelinux.erb

```diff
@@ -0,0 +1,12 @@
+<%#
+kind: PXELinux
+name: RancherOS PXELinux
+model: ProvisioningTemplate
+oses:
+- RancherOS
+-%>
+DEFAULT rancheros
+
+LABEL rancheros
+  KERNEL <%= @kernel -%> rancher.state.dev=LABEL=RANCHER_STATE rancher.state.autoformat=[/dev/xvda] rancher.cloud_init.datasources=['url:<%= foreman_url('provision')-%>']
```

Comment:
The `/dev/xvda` is still unparametrized, can you add a param and default to this when not set?

Reply:
So install-disk or autoformat_device ?

Reply:
install-disk please, that variable is used in some other templates.